src/components-v2/InfluenceVault/InfluenceVaultUtil.tsx

```diff
@@ -0,0 +1,131 @@
+import mainnetDeploy from '../../config/deployments/mainnet.json';
+
+export const parseText = (text: string[], createLink: (a: string, b: string) => JSX.Element) => {
```

Comment:
what do you think about using a parser like this https://www.npmjs.com/package/marked  and style directly the `<a></a>` directly?. This was used in the website app https://github.com/Badger-Finance/landing-page-v2/blob/main/src/components/RichText.tsx